### PR TITLE
Add new option to make a whole in the wall

### PR DESCRIPTION
**Pull request by innakoss**
Please review this, @casey-collab

### CODE REVIEW

File: ukrainian/make-a-hole/in-the-wall.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Оговтуюсь від шоку, беру виделку і виковирюю дирку у зефіровій стіні.. Що буде зі мною далі?
```
Comment (casey-collab) [inline ukrainian/make-a-hole/in-the-wall.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: ukrainian/make-a-hole/in-the-wall.md (hunk 1 above)
Comment (innakoss) [inline ukrainian/make-a-hole/in-the-wall.md]: Please review my changes, @casey-collab

**Comment by innakoss** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :sunglasses:

**Pull request merged**